pallets/frequency-tx-payment/src/benchmarking.rs

```diff
@@ -38,6 +69,104 @@ mod benchmarks {
 		Ok(())
 	}
 
+	#[benchmark]
+	fn charge_tx_payment_free() {
+		let caller: T::AccountId = whitelisted_caller();
+		<<T as pallet_transaction_payment::Config>::OnChargeTransaction as OnChargeTransaction<
+			T,
+		>>::endow_account(&caller, 100_000u64.into());
```

Comment:
Would it be possible to use frame_system instead of introducing a benchmarking feature in OnChargeTransaction to credit a balance to an account?